crates/bindings-csharp/BSATN.Runtime.Tests/BSATN.Runtime.Tests.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <RootNamespace>SpacetimeDB</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="CSharpier.Core" Version="0.28.2" />
```

Comment:
I don't think you need this and the Logging one - they are necessary for codegen tests, which use it to format snapshots, but shouldn't be necessary for runtime ones.